lib/assets/AssetsSource.js

```diff
@@ -82,4 +82,32 @@ AssetsSource.prototype.toString = function() {
   return this.srcPath + "/" + this.pattern;
 };
 
+/**
+  * Build a string suitable for caching an instance of this
+  * @returns {String} the cache key
+  */
+AssetsSource.prototype.cacheKey = function(namespace) {
+  function skipSomeKeys(key, value) {
```

Comment:
I don't believe this closes over anything useful, so lets hoist this out to the root of the module, this will prevent the necessary redefinition of `skipSomeKeys` each time `cacheKey` function is invoked